CHANGELOG/v1.6.1.md

```diff
@@ -0,0 +1,47 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.29.x
+- Workload Cluster: v1.23.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+
+* Kubernetes v1.29 is now supported
```

Comment:
i believe the `Highlights` section is only used for the minor release:
> If this is a vX.X.0 release, fill in the content for the Highlights section. Otherwise, remove the section altogether.

https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#create-pr-for-release-notes

Reply:
I think it's worth highlighting that we support 1.29

Why not? :)

Reply:
I mean if there are no highlights, sure let's skip it. But otherwise folks could only infer that we now support 1.29 by carefully reading and comparing the "Kubernetes version support" section between this and the v1.6.0 release

Reply:
sure sounds good to me 👍